ui.frontend/src/view/FormFieldBase.js

```diff
@@ -123,6 +144,53 @@ export default class FormFieldBase extends FormField {
         this.widget.value = value;
     }
 
+
+    _showHideTooltip(show) {
+        const toolTip = this.getTooltipDiv(),
+            bemBlock = this.constructor.bemBlock,
+            tooltipVisible = this._model.properties['af:layout'] ? this._model.properties['af:layout'].tooltipVisible : false,
```

Comment:
instead of directly accesing the property, access it via this._model.getState() (as done in _applyState)

Reply:
add a private function to check if tooltip is visible

Reply:
Done